com.unity.shadergraph/Documentation~/Parallax-Occlusion-Mapping-Node.md

```diff
@@ -2,30 +2,29 @@
 
 ## Description
 
-You can use the Parallax Occlusion Mapping (POM) Node to create a parallax effect that displaces a material's UVs and depth to create the illusion of depth inside that material.
+You can use the Parallax Occlusion Mapping (POM) node to create a parallax effect that displaces a material's UVs and depth to create the illusion of depth inside that material.
 
-If you experience texture sampling errors while using this node in a graph which includes Custom Function Nodes or Sub Graphs, it might be possible to resolve these errors by upgrading to version 10.3 or later.
-
-When you assign the same Texture2D to a Parallax Occlusion Mapping Node and a Sample Texture 2D Node, you need to avoid transforming the UV coordinates twice. To prevent this, connect the Split Texture Transform Node’s **Texture Only** port to the Sample Texture 2D Node’s UV port.
-
-![](images/node-parallaxocclusionmapping.PNG)
+If you experience texture sampling errors while using this node in a graph that includes Custom Function Nodes or Sub Graphs, it might be possible to resolve these errors by upgrading to Shader Graph version 10.3 or later.
 
+When you assign the same Texture2D to a POM node and a Sample Texture 2D node, you need to avoid transforming the UV coordinates twice. To prevent this, connect the Split Texture Transform node’s **Texture Only** port to the Sample Texture 2D Node’s UV port.
```

Comment:
"If you assign the same Texture2D to a POM node and a Sample Texture 2D node, connect the Split Texture Transform node's **Texture Only** port to the Sample Texture 2D node's **UV** port. This stops Unity from transforming the UV coordinates twice."

Make sure Texture Only and UV are bolded, please.

Reply:
done